.idea/misc.xml

```diff
@@ -66,7 +67,7 @@
       <textMaps />
     </LinkMapSettings>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_21" default="true" project-jdk-name="temurin-21" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_21" project-jdk-name="21" project-jdk-type="JavaSDK">
```

Comment:
Sorry, that was unwanted. Should I revert? Or maybe that means `misc.xml` should be ignored? I didn't have a `temurin-21` installed

Reply:
Don't worry about that, it should probably be ignored. Same with `.idea/jsLibraryMappings.xml` and  `.idea/kotlinc.xml`. I wonder why these are not mentioned in [JetBrains docs](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems).

I'll take care of this by expanding `.gitignore`.